include/rocksdb/options.h

```diff
@@ -2057,12 +2057,19 @@ struct ReadOptions {
 
   // *** BEGIN options for RocksDB internal use only ***
 
+  // TODO: create a new struct for per-request options, potentially including
+  // timestamps in point lookups/scans
+
+  // request_id a unique id assigned by the application. It is used to allow us
+  // to link file system metrics/logs to rocksDB and application logs.
+  const std::string* request_id = nullptr;
```

Comment:
Its not for RocksDB internal use only, so I suggest defining a new section for options that may be moved to a new struct at some point in the future.